docs/source/configuration.rst

```diff
@@ -283,23 +283,15 @@ Cluster configuration
 
     Type of GCP or AWS machine to start as worker node(s). 
 
+    **WARNING**: ElasticBLAST will select a machine type for you with sufficient RAM to hold your database in memory if you search an NCBI provided database or provide metadata for your custom database (see :ref:`tutorial_create_blastdb_metadata`).  This is the recommended way to use ElasticBLAST.  Specifiying the machine type will override this feature, and you need to be sure that your machine type has sufficient memory to hold you database.  
+
     **NOTE**: The machine's available RAM should be large enough to contain the sequences in the database (one byte per residue or one byte per four bases) plus ~20%.
 
     * Default: ``n1-standard-32`` for GCP, ``m5.8xlarge`` for AWS.
```

Comment:
@tom6931 , per @boratyng 's comment, it looks like the default should be:

```suggestion
    * Default: ``n1-highmem-32`` for GCP, ``m5.8xlarge`` for AWS.
```

I corroborated this in the code: https://github.com/ncbi/elastic-blast/blob/master/src/elastic_blast/constants.py#L101